app/assets/stylesheets/_base.scss

```diff
@@ -0,0 +1,170 @@
+body {
+  background-color: #fff;
+  border-top: none;
+  display: flex;
+  flex-direction: column;
+  font-family: "proxima-nova-1", "proxima-nova-2", "museo-sans-1", "museo-sans-2", helvetica neue, sans-serif;
+  line-height: 20px;
+  margin: 0 auto;
+  min-height: 100vh;
+  min-width: 320px;
+}
+
+img {
+  max-width: 100%;
+}
+
+section {
+  flex: 1;
+}
+
+.content {
+  /*disclude for now */
+  /*@include outer-container;*/
+  max-width: 1100px;
+  flex: 1;
+  padding: 1.5em 1em 3em;
+  width: 100%;
+
+  ul, ol {
+    padding: 0;
+  }
+
+  li {
+    margin-top: 1rem;
+
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+
+  .text-box-wrapper {
+    /*@include span-columns(8.5);*/
+    margin-right: 3%;
+    position: relative;
+
+/*    &:before {
+      @include position(absolute, 10px -35px 0px null);
+      content: "";
+
+      @include dashboard-small-only {
+        border-right: 0;
+      }
+    }*/
+
+    &.solo {
+      margin: auto;
+      float: none;
+    }
+
+    &.right {
+      float: right;
+      margin: 0 0 0 flex-gutter(12);
+    }
+  }
+
+  .summary {
+    clear: both;
+    display: block;
+    float: left;
+    /*@include span-columns(9);*/
+    text-align: left;
+
+    p {
+      margin: 0 0 .4rem 26px;
+    }
+
+    p:last-child {
+      margin-bottom: 2rem;
+    }
+  }
+}
+
+aside {
+  /*@include span-columns(3);*/
+  box-shadow: none;
+  display: inline-block;
+  float: right;
+  font-size: 0.96rem;
+  margin-bottom: 1em;
+  padding-left: 30px;
+  position: relative;
+
+  hgroup {
+    margin-left: 48px;
+
+    h5, h4 {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line